tests/test-cases/Group1-Docker-Commands/1-43-Docker-CP-Offline.robot

```diff
@@ -0,0 +1,274 @@
+    Should Be Equal As Integers  ${rc}  0
+    Should Not Contain  ${output}  Error
+
+Concurrent copy: create processes to copy a small file from host to offline container
+    ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} create -i --name offline -v vol1:/vol1 ${busybox}
+    Should Be Equal As Integers  ${rc}  0
+    Should Not Contain  ${output}  Error
+    ${pids}=  Create List
+    Log To Console  \nIssue 10 docker cp commands for small file
+    :FOR  ${idx}  IN RANGE  0  10
+    \   ${pid}=  Start Process  docker %{VCH-PARAMS} cp ${CURDIR}/foo.txt offline:/foo-${idx}  shell=True
+    \   Append To List  ${pids}  ${pid}
+    Log To Console  \nWait for them to finish and check their RC
+    :FOR  ${pid}  IN  @{pids}
+    \   Log To Console  \nWaiting for ${pid}
+    \   ${res}=  Wait For Process  ${pid}
+    \   Log To Console  ${res.stdout}
+    \   Should Be Equal As Integers  ${res.rc}  0
+    ${output}=  Start container and run command  offline  ls /
+    Log To Console  \nCheck if the copy operations succeeded
+    :FOR  ${idx}  IN RANGE  0  10
+    \   Should Contain  ${output}  foo-${idx}
+    ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} stop offline
+    Should Be Equal As Integers  ${rc}  0
+    Should Not Contain  ${output}  Error
+
+Concurrent copy: repeat copy a large file from host to offline container several times
+    ${pids}=  Create List
+    Log To Console  \nIssue 10 docker cp commands for large file
+    :FOR  ${idx}  IN RANGE  0  10
+    \   ${pid}=  Start Process  docker %{VCH-PARAMS} cp ${CURDIR}/largefile.txt offline:/vol1/lg-${idx}  shell=True
+    \   Append To List  ${pids}  ${pid}
+    Log To Console  \nWait for them to finish and check their RC
+    :FOR  ${pid}  IN  @{pids}
+    \   Log To Console  \nWaiting for ${pid}
+    \   ${res}=  Wait For Process  ${pid}
+    \   Log To Console  ${res.stdout}
+    \   Should Be Equal As Integers  ${res.rc}  0
+    ${output}=  Start container and run command  offline  ls /vol1
+    Log To Console  \nCheck if the copy operations succeeded
+    :FOR  ${idx}  IN RANGE  0  10
+    \   Should Contain  ${output}  lg-${idx}
+    ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} stop offline
+    Should Be Equal As Integers  ${rc}  0
+    Should Not Contain  ${output}  Error
+
+Concurrent copy: repeat copy a large file from offline container to host several times
+    ${pids}=  Create List
+    Log To Console  \nIssue 10 docker cp commands for large file
+    :FOR  ${idx}  IN RANGE  0  10
+    \   ${pid}=  Start Process  docker %{VCH-PARAMS} cp offline:/vol1/lg-${idx} ${CURDIR}  shell=True
+    \   Append To List  ${pids}  ${pid}
+    Log To Console  \nWait for them to finish and check their RC
+    :FOR  ${pid}  IN  @{pids}
+    \   Log To Console  \nWaiting for ${pid}
+    \   ${res}=  Wait For Process  ${pid}
+    \   Log To Console  ${res.stdout}
```

Comment:
don't log this to console, just Log this otherwise it is not easy to debug after the fact